plugins/core/base/routes/tool-default.js

```diff
@@ -86,10 +99,22 @@ module.exports = {
             tool: Joi.string().required(),
             path: Joi.string().required()
           },
-          payload: Joi.object()
+          query: {
+            appendItemToPayload: Joi.string().optional(),
+          },
+          payload: Joi.object(),
+          options: {
+            allowUnknown: true
+          }
         }
       },
       handler:  async (request, h) => {
+        if (request.query.appendItemToPayload) {
+          const item = await request.server.methods.db.item.getById(request.query.appendItemToPayload);
+          request.payload = {
```

Comment:
should the payload of the request really be overwritten by item? wouldn't be better to just add the item to the payload?

Reply:
no, that's a mistake that is just fixed with the next commit.

Reply:
maybe a test could be added for that case?

Reply:
yes :-) will do